tests/testing_mesh_creation_functions.py

```diff
@@ -737,6 +738,99 @@ def line(t):
         # Check the output.
         compare_test_result(self, input_file.get_string(header=False))
 
+    def test_mesh_creation_functions_helix_no_rotation(self):
+        """Create a helix and compare it with the reference file."""
+
+        # Create mesh.
+        mesh = Mesh()
```

Comment:
For such simple test cases you don't need to define the mesh, simply pass `input_file` to `create_beam_mesh_helix` (also for the other test cases)